src/main/java/com/app/onlinebookstore/dto/shoppingcart/UpdateCartItemDto.java

```diff
@@ -0,0 +1,12 @@
+package com.app.onlinebookstore.dto.shoppingcart;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+
+@Data
+public class UpdateCartItemDto {
+    @NotNull
+    @Min(0)
+    private Integer quantity;
```

Comment:
maybe it's a better way to add @Positive annotation only or @PositiveOrZero instead of these two?

Reply:
I don't think that this has big sence